projects/packages/videopress/src/class-wpcom-rest-api-v2-endpoint-videopress.php

```diff
@@ -179,6 +179,17 @@ public function videopress_block_update_meta( $request ) {
 					);
 				}
 
+				if ( isset( $json_params['description'] ) ) {
+					$meta['videopress']['description'] = sanitize_text_field( $json_params['description'] );
```

Comment:
```suggestion
					$meta['videopress']['description'] = sanitize_textarea_field( $json_params['description'] );
```
This should probably use [sanitize_textarea_field](https://developer.wordpress.org/reference/functions/sanitize_textarea_field/)